src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Deposit/DepositCommon.ts

```diff
@@ -69,3 +69,5 @@ export type ThorchainSaversDepositActions =
   | SetLoading
   | SetTxid
   | SetIsExactAllowance
+
+export const RUNEPOOL_DEPOSIT_MEMO = 'POOL+'
```

Comment:
preferably-blocking: handle me in `useSendThorTx` and add `depositRunePool` to the `Action` memo

Reply:
I don't get this?, `depositRunePool` wouldn't be used at all for now and `useSendThorTx` doesn't seem responsible of initiating the memo at all

Reply:
I did move it to a thorchain constant file in [19db6af](https://github.com/shapeshift/web/pull/7440/commits/19db6afed2d77fec1f3cb46386d3487d42dbb41e)